rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -24,6 +26,8 @@ The final phase in the life-cycle of a process is the *Production* phase. This p
 * If you want to use the values of xref:processautomation-prepare-deployment-variable.adoc[global variables] or xref:processautomation-prepare-deployment-credential.adoc[credentials] as initial values for activity parameters, an RPA administrator must create them in the *Global Variables* or *Credential Pool* view first.
 * A project in test or production phase must exist in the *Project Management* view of the *Process Automation* module.
 * Obtain the needed permissions as a member of the process team in the project data sheet in the *Project Management* view of the *Process Automation* module.
+* To run a process in an existing console or RDP session, you must deploy it to a bot that is configured for console sessions and the specified user must be logged in in this session when you deploy the run configuration. If the user is logged in more than once, the bot starts the process in the first session it finds the user logged in to. The bot searches for the user in the order real console session, RDP session 1 to n. Do not close or minimize the session while the bot executes the process.
+* To run a process in a hidden session, you must deploy it to a bot configured for secure sessions. Before deploying a run configuration on a bot that has only one secure session licensed, do not disconnect the session, but sign out the specified user. Do not sign in to a computer where a bot with one licensed secure session is running a process.
```

Comment:
I suggest reversing the sign-out text and the "do not disconnect the session" text. I always think it's better to put the positive text first:

"...sign out the specified user but do not disconnect the session."

Reply:
I left this as was because disconnecting by clicking *Disconnect* does not sign out the user. This is a bad habit of our users that leads to difficult to trace errors in running a process.